arduino/include/utils.h

```diff
@@ -1,11 +1,11 @@
 template <typename T>
-void serialPrintLn(const T message) {
-    Serial.println(message);
+void serialPrintLn(const T &message) {
+    // Serial.println(message);
 }
 
 template <typename T, typename... Types>
-void serialPrintLn(const T message, Types... otherMessages) {
-    Serial.print(message);
+void serialPrintLn(const T &message, Types... otherMessages) {
+    // Serial.print(message);
```

Comment:
Strings ended up taking way too much place, causing crashes (yes, apparently an Arduino can crash!). The compiler is smart enough to not include the strings when those lines are commented. A future PR will bring a proper error message system that is space-efficient.